manager/system_handler.py

```diff
@@ -145,6 +145,8 @@ def _id_query(rh_account_id, query_args):
                         SystemVulnerabilities.status_id,
                         SystemVulnerabilities.status_text,
                         InsightsRule.name.alias('rule_id'),
+                        fn.COALESCE(CveAccountData.status_id, 0).alias('cve_status_id'),
```

Comment:
I know you probably seen it in other code snippets, but `status_id` is no longer a nullable field. Thats why the `COALESCE` is not needed. 
```
                Table "public.cve_account_data"
       Column       |  Type   | Collation | Nullable | Default 
--------------------+---------+-----------+----------+---------
 cve_id             | integer |           | not null | 
 rh_account_id      | integer |           | not null | 
 business_risk_id   | integer |           | not null | 0
 business_risk_text | text    |           |          | 
 status_id          | integer |           | not null | 0
 status_text        | text    |           |          | 
```

Reply:
Oh sorry, I just noticed that there is a `LEFT JOIN`, nevermind :D